frontend/src/static/js/v_summary.js

```diff
@@ -168,7 +168,8 @@ window.vSummary = {
     getFilterHash() {
       const { addHash } = window;
 
-      this.filterSearch = this.filterSearch.toLowerCase();
+
+      this.filterSearch = this.filterSearch;
```

Comment:
This line doesn't do anything anymore right?

Reply:
Yes it is a redundant line, will remove it.